src/AppKit/NSAccessibility.cs

```diff
@@ -143,7 +144,7 @@ public static string GetRoleDescription (NSString role, NSString subrole)
 				subroleHandle = subrole.Handle;
 
 			IntPtr handle = NSAccessibilityRoleDescription (role.Handle, subroleHandle);
-			return NSString.FromHandle (handle);
+			return CFString.FromHandle (handle);
```

Comment:
Wild idea: if NSString and CFString are toll-free bridged, then can't we just change the implementation of `NSString.FromHandle` to call `CFString.FromHandle`?

Reply:
Yeah, I thought about that one as it would have required less code changes.
Somehow it feels like cheating... because it would not do what you would expect (even if the result is the same).
Maybe we could add an `[Obsolete]` on `NSString.FromHandle` ?

Reply:
Yeah, I agree with Sebastien, having `CFString.FromHandle` is cleaner and clear on the intention, not sure about the obsolete, maybe better adding an Advice?

Reply:
> Somehow it feels like cheating...

That's why I called it a wild idea, it feels wrong to me too :smile:

> Maybe we could add an `[Obsolete]` on `NSString.FromHandle` ?
> maybe better adding an Advice?

I don't lean particularly in any way, I think either would work just fine.

Reply:
@dalexsoto `[Advice]` might not show up (without additional work in both IDE) on .net6

However `[Obsolete]` has been enhanced so you can now give them error codes (to ignore warnings or make them errors) and you can supply an URL for more information. That seems to cover (and more) what we wanted to `[Advice]`

Reply:
done (adding `[Obsolete]`)

Reply:
> [Advice] might not show up (without additional work in both IDE) on .net6

The analyzers can be made into non-IDE analyzers

Reply:
> The analyzers can be made

Yes, but key words are: _without additional work_. We already have a bunch of analyzers that requires more work for net6.

> nto non-IDE analyzers

The goal of `[Advice]` was to give IDE-only hints to developers.